Plugin/src/main/kotlin/com/shopify/testify/TestifyExtension.kt

```diff
@@ -95,6 +96,9 @@ internal data class TestifySettings(
         if (extension.installAndroidTestTask?.isNotEmpty() == true) {
             this.installAndroidTestTask = extension.installAndroidTestTask
         }
+        if (extension.autoImplementLibrary != null) {
```

Comment:
is this even possible?

Reply:
for it to be `null` i mean

Reply:
Yes, it's possible

There are two classes that work together in this code. The Setting and the Extension
The Extension is a gradle concept. It looks like `testify {}` in your build.gradle file.
In the Extension (which this line is checking) everything can be null. In this context, null means "not specified"

The Settings are a concept unique to Testify. It's a cache of the Extension values, fully resolved and with all defaults applied as necessary.

In this block, we're checking to see if the `autoImplementLibrary` extension is `not specified` (null) or some value.
If it's a some value, we override Settings to that.